tests/playwright/utilities/terminal.js

```diff
@@ -0,0 +1,36 @@
+import { execSync } from 'node:child_process';
+import path from 'path';
+
+class Terminal {
```

Comment:
```suggestion
class Terminal {
```
export default class Terminal {

Reply:
Maybe we can only do this when these scripts have been refactored to TypeScript